### PR TITLE
Migrate ZPagesExporter to BatchExportActivityProcessor

### PR DESCRIPTION
**Pull request by eddynaka**
Related to #1078.

## Changes

Please provide a brief description of the changes here. Update the
`CHANGELOG.md` for non-trivial changes.

For significant contributions please make sure you have completed the following items:

* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Exporter.ZPages/ZPagesExporterHelperExtensions.cs [hunk 1]
```diff
@@ -45,7 +45,7 @@ public static class ZPagesExporterHelperExtensions
             var zpagesExporter = new ZPagesExporter(exporterOptions);
 
             // TODO: Pick Simple vs Batching based on ZipkinExporterOptions
-            return builder.AddProcessor(new SimpleActivityProcessor(zpagesExporter));
+            return builder.AddProcessor(new BatchExportActivityProcessor(zpagesExporter));
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.ZPages/ZPagesExporterHelperExtensions.cs:48]: I guess ZPages can use the simple processor?

Comment (eddynaka) [reply src/OpenTelemetry.Exporter.ZPages/ZPagesExporterHelperExtensions.cs:48]: i think so. I will close this pr then.

Comment (CodeBlanch) [reply src/OpenTelemetry.Exporter.ZPages/ZPagesExporterHelperExtensions.cs:48]: If you look below, it seems to have its own processor?

Comment (reyang) [reply src/OpenTelemetry.Exporter.ZPages/ZPagesExporterHelperExtensions.cs:48]: IMHO ZPages is not an exporter at all, it just a processor... 😆

Comment (CodeBlanch) [reply src/OpenTelemetry.Exporter.ZPages/ZPagesExporterHelperExtensions.cs:48]: This is honestly the first time I have ever looked at the ZPages code but I think you are correct. This seems like a bug:

`return builder.AddProcessor(new SimpleActivityProcessor(zpagesExporter));`

Should be as you have above I think. Just 1 call to AddZPagesExporter which does AddProcessor(new ZPagesProcessor(new ZPagesExporter))?

Now there are probably also some bigger design issues at play here. Should all of this logic really be split across two things? Maybe it should all be in the processor or all in the exporter? Not sure 😄

Comment (eddynaka) [reply src/OpenTelemetry.Exporter.ZPages/ZPagesExporterHelperExtensions.cs:48]: i will push this change doing this refactor, but maybe should be a good idea to check this "exporter"

**Comment by cijothomas** [comment]
@eddynaka can you rsolve conflict?

**Comment by eddynaka** [comment]
> @eddynaka can you rsolve conflict?

yes, give me some minutes!

**Comment by eddynaka** [comment]
> @eddynaka can you rsolve conflict?

just updated! CI running!

**Pull request merged**